.github/scripts/validate_binaries.sh

```diff
@@ -39,6 +39,12 @@ else
         ${PWD}/check_binary.sh
     fi
 
+    # NB: The latest conda 23.11.0 pulls in some dependencies of conda-libmamba-solver that
+    # require GLIBC_2.25, which is not available in the current aarch64 image causing the
+    # suqsequence git command to fail. Basically, they don't work with CentOS 7 which AML 2
+    # is based on https://github.com/ContinuumIO/anaconda-issues/issues/12822
+    unset LD_LIBRARY_PATH
+
```

Comment:
Maybe better approach here would be to 
- Save path
- Unset
- Set it back before finishing

Reply:
Also please add check for
https://github.com/pytorch/builder/blob/main/.github/workflows/validate-aarch64-linux-binaries.yml#L92